services/movie_session.py

```diff
@@ -0,0 +1,72 @@
+from db.models import MovieSession, Movie, CinemaHall
+from django.db.models import QuerySet
+
+
+def create_movie_session(
+    movie_show_time: str, movie_id: int, cinema_hall_id: int
+) -> MovieSession:
+    """
+    Create a movie session with the provided parameters.
+    """
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```

Comment:
Consider handling exceptions for `Movie.objects.get(id=movie_id)` and `CinemaHall.objects.get(id=cinema_hall_id)` to avoid potential `DoesNotExist` errors if the IDs are invalid.